docs/pages/about/about_ro_crate.md

```diff
@@ -12,107 +12,108 @@ sidebar: about
 1. TOC 
 {:toc}
 
+## About RO-Crate
```

Comment:
This is already the top heading, so doesn't need duplicating

```suggestion
```

Reply:
It is in the top heading. However I think it looks a little strange to go straight from the TOC to content without a separating section title.

Perhaps we could use a different title here: 'FAIR metadata'?

Reply:
'Purpose of RO-Crate' perhaps ?

Reply:
How about 'Why use RO-Crates'?

Reply:
Ah, let's just leave it as is 😆